packages/interactor/src/integration/cypress.ts

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { bigtestGlobals } from '@bigtest/globals';
+
+declare const Cypress: any;
```

Comment:
if you add 
```ts
 "types": ["cypress"]
```

Then you can delete these `any` declarations and get full cypress support

Reply:
Oh thank you. It looks like I had to add cypress as a devDependency for `"types: ["cypress"]` to work but I'm guessing that's fine.